spec/factories/invoices.rb

```diff
@@ -0,0 +1,15 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :invoice do
+    association :customer # automatically create a customer record when an invoice is created using factory bc of the foreign key association
+
+    status { Invoice.statuses.keys.sample } # Randomly pick one of the status keys from enum
```

Comment:
I created a faker for invoices as well, but for status I used `status {rand(0..2)}`. Either should work; let's go with yours since you beat me to the PR! Also if we update the number of statuses for some reason yours will follow, where my version would have to be caught and changed.

Reply:
Updated `status { rand(0..2) }` since both you and Kevin are using it! The reason I chose Invoice.statuses.keys.sample is because it's often best practice to not hard code and to be dynamic just in case something changes later on and we don't have to worry about everything breaking